requirements.txt

```diff
@@ -1,5 +1,5 @@
 asyncpg
-discord.py[voice]
+discord.py[voice]==1.4.1
```

Comment:
Right now this will cause problems with the intents code in `dozer/__main__.py`, as that only exists on 1.5+. Maybe this should be a `>=` constraint?

Reply:
Or at least a version bump? We probably ought to do that anyway